packages/data-point/lib/reducer-types/reducer-helpers/index.js

```diff
@@ -0,0 +1,68 @@
+const createStub = require('./reducer-stub').create
+
+const reducerAssign = require('./reducer-assign')
+const reducerFilter = require('./reducer-filter')
+const reducerFind = require('./reducer-find')
+const reducerMap = require('./reducer-map')
+const reducerOmit = require('./reducer-omit')
+const reducerPick = require('./reducer-pick')
+
+/*
+"Reducer helpers" are just functions that create reducers.
+
+They work by generating a temporary object called a
+ReducerStub that is later compiled into a reducer:
+
+// first, the user creates a ReducerStub
+
+const reducer = ReducerStub('filter', '$a.b')
+
+// when the stub is passed to transform, it will
+// be compiled into an actual reducer object
+
+dp.transform(reducer, data)
+
+The helpers exposed on DataPoint are just the
```

Comment:
Again, please remove "just".

Also, this comment makes it seem like we shouldn't call it "helpers" if there is a single function being exposed. 🤔

Reply:
1. done

2. There are multiple functions exposed, like `DataPoint.helpers.map`, `DataPoint.helpers.find`, etc, but I'm fine with changing the name if you have any suggestions.

Reply:
> There are multiple functions exposed, like `DataPoint.helpers.map`, `DataPoint.helpers.find`, etc, but I'm fine with changing the name if you have any suggestions.

OK, that's what I thought we were doing, but got confused by the comment I guess. 🙃

Thanks for clarifying that for me.